### PR TITLE
Added params for Vlan Networking

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

**Comment by sghai** [comment]
APT.

**Comment by oshtaier** [comment]
Can you explain more why we need these settings and where they will be used?

**Comment by sghai** [comment]
These settings are required for host provisioning and discovery. We will use this info to setup provisioning and host discovery. See this issue for more info #3063

**Comment by oshtaier** [comment]
Thanks @sghai for answer. Now I see it is a part of complex issue that run away from my attention

File: robottelo.properties.sample [hunk 1]
```diff
@@ -122,6 +122,13 @@ ssh_key=
 # insights_client_el6repo=https://www.example.com/insights-client/repo/insights-client-6.repo
 # insights_client_el7repo=https://www.example.com/insights-client/repo/insights-client-7.repo
 
+# VLAN Networking details
```
Comment (oshtaier) [inline robottelo.properties.sample]: Please, put more details to that description

**Comment by oshtaier** [comment]
ACK non blocking comment

File: robottelo.properties.sample (hunk 1 above)
Comment (kbidarkar) [inline robottelo.properties.sample]: Ok, will update with the info provided by @sghai.

File: robottelo/config/settings.py [hunk 2]
```diff
@@ -263,6 +263,32 @@ def validate(self):
         return validation_errors
 
 
+class VlanNetworkSettings(FeatureSettings):
```
Comment (elyezer) [inline robottelo/config/settings.py]: Can you move this class definition to match the alphabetical order? All other config classes are following this.

**Comment by elyezer** [comment]
Missing an entry on Settings class for the feature.

ACK, pending comments.

File: robottelo/config/settings.py (hunk 2 above)
Comment (kbidarkar) [inline robottelo/config/settings.py]: Oh! ok.

**Comment by oshtaier** [comment]
ACK

**Comment by sthirugn** [comment]
:+1:

**Comment by elyezer** [comment]
Merging without the entry on main settings. @sghai will work on adding the entry.

**Pull request merged**